### PR TITLE
gp updated readme documentation.

### PR DESCRIPTION
**Pull request by germainep**
removed placeholder data and updated API description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,37 +35,44 @@ Add contributor info below, make sure add images and edit the social links for e
...
+### Get all work odrers
+ - URL: /orders/
+ - Method: GET
+### Get a work order by Id
+ - URL: /orders/:id
+ - Method: GET
 
-This is the way you take out your flustrations. Get away from those little Christmas tree things we used to make in school. Isn't it fantastic that you can change your mind and create all these happy things? Everything's not great in life, but we can still find beauty in it.
+### Create a new work order
+ - URL: /oders
```
Comment (tommyconner96) [inline README.md]: Noticed a typo here! everything else looks good

**Review by tommyconner96 — CHANGES_REQUESTED** [review]
Besides that small typo, looks great

File: README.md (hunk 1 above)
Comment (germainep) [reply README.md]: fixed

**Review by tommyconner96 — APPROVED** [review]

**Review by sophiasagan — APPROVED** [review]
Looks good

**Pull request merged**